components/Header/Header.tsx

```diff
@@ -0,0 +1,23 @@
+import Image from 'next/image';
+import LogoImageMobileSrc from '../../public/images/MINI_LOGO.svg';
+import LogoImageDesktopSrc from '../../public/images/LOGO.svg';
+
+export function Header() {
+  return (
+    <header className="header">
+      <div className="container header__wrapper">
+        <Image
+          src={LogoImageMobileSrc}
+          className="header__logo header__logo--small"
+          alt="headerLogo"
```

Comment:
Let's avoid camel case in alts, e.g. here it can be "Header logo". Or I would call it "Best By Logo"